### ISSUE TITLE
Add an example of code without pathify vs with pathify

### ISSUE THREAD

**Issue by hitautodestruct** [davestewart/vuex-pathify#66]
Coming in from heavy use of vuex in an app I'm working on it took me a while to understand why I needed pathify.

I found it very hard to understand how pathify reduces my code until I found [this post](https://alligator.io/vuejs/vuex-pathify/) comparing regular vuex methods to pathify.

I would suggest you add at least one such example on the pathify [101 page](https://davestewart.github.io/vuex-pathify/#/intro/pathify) or a similar introductory page.

**Comment by davestewart** [comment]
That's a good suggestion.

I've really struggled with keeping the docs to a minimum on this lib, but I think that could be helpful.

There are examples dotted throughout the docs, but you're looking for just a single place, perhaps:

- intro / vuex comparison

?

**Comment by hitautodestruct** [comment]
Yes, although it could just be one direct comparison on the pathify 101 page.

I found some direct comparisons of pathify vs vuex methods on the [components helpers page](https://davestewart.github.io/vuex-pathify/#/api/component?id=sync).

Basically I was looking for the equivalent vuex code, because thats what I know and where I come from.
Once, I saw the comparison it was clear what pathify does.

**Comment by WhiteBookmark** [comment]
Shouldn't this be closed ? I already see examples of vuex equivalent code in docs.

**Comment by hitautodestruct** [comment]
@WhiteBookmark Yes, if you dig deep you will see examples.
I am trying to make the point that initially it is very unclear as to what pathify gives you when readin g the intro and 101.

**Comment by davestewart** [comment]
I'm happy to add another section with some basic examples.

It's really hard to get the level of detail right in the docs. I always aim to keep them lean, but always end up writing much more than I would like to have, hence it becomes harder to find things.

In an ideal world, I'd make the docs leaner, but I doubt that will happen, so this is a good compromise.

**Comment by julie777** [comment]
Keeping the docs simple and short is admirable. However as a new user exploring, it is verying important that the first thing they read in the readme on GitHub or on NPM includes enough to make them think it will be *useful and worthwhile* for them. I am all for adding a simple "pathify lets you go from ... to ... and that is why you should try it. Seeing the actual code change it *huge*.

For me seeing what ```<script setup>``` does in Vue 3 made me an instant convert. I hate having to sift through a bunch of code details to extract the meaning, especially when there is a simple way that makes the intent more obvious.
